draft-ietf-quic-qlog-main-schema.md

```diff
@@ -1514,86 +1514,70 @@ still provide adequate output for incomplete logs.
-  !RFC9000}}).
+* Session, Connection, or User identifiers e.g., QUIC Connection IDs {{Section
+  9.5 of !RFC9000}}).
 
-* System-level information such as CPU, process, or thread identifiers.
+* System-level information e.g., CPU, process, or thread identifiers.
 
-* Stored State which can be used to correlate individual connections or sessions
-  over time. Examples include QUIC address validation and retry tokens, TLS
-  session tickets, and HTTP cookies.
+* Stored State e.g., QUIC address validation and retry tokens, TLS session
+  tickets, and HTTP cookies.
 
-* Decryption keys, passwords, and tokens which can be used with other data
-  sources (e.g., captures of encrypted packets) to correlate qlog data to a
-  specific connection or user or leak additional information. Examples include
-  TLS decryption keys and HTTP-level API access or authorization tokens.
+* TLS decryption keys, passwords, and HTTP-level API access or authorization tokens.
 
-* Data that can be used to correlate qlogs to other data sources (e.g., captures
-  of encrypted packets). Examples include high-resolution event timestamps or
-  inter-event timings, event counts, packet and frame sizes.
+* High-resolution event timestamps or inter-event timings, event counts, packet
+  sizes, and frame sizes.
 
-* Full or partial encrypted raw packet and frame payloads, which can be used
-  with other data sources (e.g., captures of encrypted packets) to correlate
-  qlog data to a specific connection or session.
+* Full, or partial encrypted, raw packet and frame payloads.
 
-* Full or partial plaintext raw packet and frame payloads (e.g., HTTP Field
-  values, HTTP response data, TLS SNI field values), which can contain directly
-  sensitive information.
+* Full, or partial, plaintext raw packet and frame payloads e.g., HTTP Field
+  values, HTTP response data, TLS SNI field values.
 
 ## Operational implications and recommendations
 
-Considerations for operational implications should focus on log capture and
-access. Logging of Internet protocols is equivalent to access to store or read
-plaintext communications, so security considerations apply similarly to qlog.
+Operational considerations should focus on authorizing capture and access to logs. Logging of
+Internet protocols using qlog can be equivalent the ability to store or read plaintext
+communications. Without a more detailed analysis, all of the security considerations of plaintext access apply.
 
-It is recommended that the ability to enable qlog capture is subject to access
-control and auditing. Furthermore, access control policy should extend to the
-level of information that is allowed to be captured (e.g., capture of
-more-sensitive data requires higher privileges).
+It is recommended that qlog capture is subject to access control and auditing,
+extending to granularity over the level of information that is allowed to be
+captured (e.g., capture of more-sensitive data requires higher privileges).
```

Comment:
`extending to granularity over the level of information that is allowed to be
captured` 

this is a very weird sentence to me that I'm having issues parsing the meaning of... could do with a re-formulation (the e.g., helps, but still :))